src/pages/interviews/serverlesschats.mdx

```diff
@@ -17,3 +17,105 @@ On this episode, Jeremy and Rebecca chat with Swizec Teller about how to approac
 ></iframe>
 
 [Serverless Chats ep #135](https://www.serverlesschats.com/135/)
+
+## Talking points
+
+- If you can JavaScript, you can back end
+- The future belongs to full stack front end engineers
+- Modern backend is a JavaScript function
+- JavaScript is a great language and anyone who can code JavaScript can pick up any other language
+- A lot of what used to make JavaScript really ugly was interacting with the Dom
+- Front end is more like a full fledged app
+- GraphQL is kind of an escape for backend engineers
+- How should a full stack or how should a front end engineer approach serverless?
+- Database and Variable Naming Conventions: Snake Case vs Camel Case
+- [Serverless Handbook](https://serverlesshandbook.dev) for Front End Engineers
+- Architecture principles and how to create good serverless developer experience
+- Your average tutorial can make you a mediocre engineer, how to break out of that cycle
+
+## Quotable Quotes
+
+"I never want to touch an Apache config ever again" - ST
+"There was this JAMstack revolution that happened over the last couple of years" - ST
+"we've now had react for almost 10 years and everyone's still happy with react, which is honestly a huge accomplishment for the JavaScript ecosystem" - ST
+"one of the coolest things about JavaScript programmers is that I feel like JavaScript is relatively easy to get started with, especially since we have ESX and modern JavaScript, is surprisingly, it's actually a very reasonable language" - ST
+"JavaScript is kind of a lot like in the English language where it mugs programming languages in dark alleys and picks off all of the best stuff from their pockets and adds it to JavaScript." - ST
+
+"JavaScript is the only language, that I know of at least, where every device on the planet has a JavaScript execution environment." - ST
+"So you can write code for literally anything, like Android can run JavaScript. iOS can run JavaScript. A lot of people don't know this but your TV set top box is actually a web browser." - ST
```

Comment:
```suggestion
"I never want to touch an Apache config ever again" - ST

"There was this JAMstack revolution that happened over the last couple of years" - ST

"we've now had react for almost 10 years and everyone's still happy with react, which is honestly a huge accomplishment for the JavaScript ecosystem" - ST

"one of the coolest things about JavaScript programmers is that I feel like JavaScript is relatively easy to get started with, especially since we have ESX and modern JavaScript, is surprisingly, it's actually a very reasonable language" - ST

"JavaScript is kind of a lot like in the English language where it mugs programming languages in dark alleys and picks off all of the best stuff from their pockets and adds it to JavaScript." - ST

"JavaScript is the only language, that I know of at least, where every device on the planet has a JavaScript execution environment." - ST

"So you can write code for literally anything, like Android can run JavaScript. iOS can run JavaScript. A lot of people don't know this but your TV set top box is actually a web browser." - ST
```